src/Compiler/test/Microsoft.NET.Sdk.Razor.SourceGenerators.Tests/TestFiles/RazorSourceGeneratorCshtmlTests/CssScoping/Pages_Index.html

```diff
@@ -0,0 +1,29 @@
+﻿<h1 test-css-scope>heading</h1>
+<header>header</header>
+<thead>table head</thead>
+<form test-css-scope>
+    <select test-css-scope>
+        <option test-css-scope>choose</option>
+    </select>
+</form>
+< test-css-scope div />
+<div test-css-scope
+>multiline</div>
+< test-css-scopediv>bang</div>
+<input test-css-scope multiple value="test" disabled="disabled" />
+<div test-css-scope></div>
+<!-- <div>HTML comment</div> -->
+    <div test-css-scope>code block</div>
+<div test-css-scope>C# template</div>
+<div test-css-scope attr="1"></div>
+<div@x test-css-scope="1"></div>
```

Comment:
Is this line correct?  

Should it not be something like:
`<div test-css-scope @x="1"></div>`

Reply:
Yes, it's probably a bug, unrelated to this PR, though. And the source doesn't look like supported Razor syntax, so I guess it doesn't really matter what is emitted (except maybe an error should be reported).

```razor
<div@x="1"></div>
```